components/tabs/tab-internal.js

```diff
@@ -95,7 +95,6 @@ class Tab extends RtlMixin(LitElement) {
 		super();
 		this.ariaSelected = 'false';
 		this.role = 'tab';
-		this.tabIndex = -1;
```

Comment:
I opted to handle tabindex management in the `tabs` component itself, including tracking its status in the `tabInfo` objects. I'm not tied to that idea; it just seemed cleaner. I'm happy to manage the tabindex within `tab-internal` if that's more desirable.

Reply:
It is possible that tabs can be added and removed afterwards and we should account for those scenarios.  For that reason, it still may be worthwhile initializing the `tabindex="-1"` here.  Also, if the selected tab is removed, a new tab must be selected... and we'd expect its `tabindex` to then get set to 0.